src/concolic/concolic_value.ml

```diff
@@ -0,0 +1,462 @@
+module type T = sig
+  type t
+end
+
+type ('c, 's) cs =
```

Comment:
Could you find a better name ? :sweat_smile: When reading `concolic_choice` it was hard for me to understand.

Reply:
Yes, certainly. I'm not sure it is really better than a pair anyway.

Reply:
I changed the name of the fields